docs/getting-started/user-guides/windows.md

```diff
@@ -6,6 +6,11 @@
 
 > Step One
 
+Paymetheus Users  
+You will already have the dcrd program installed as part of the Paymetheus package.
```

Comment:
I am almost certain this is not the markdown you wanted here because it will contain one paragraph starting with "Paymetheus Users You will..."